pyproject.toml

```
@@ -22,7 +22,7 @@ name = "lilio"
 description = "python package for generating calendars for machine learning timeseries analysis."
 readme = "README.md"
 license = "Apache-2.0"
-requires-python = ">=3.8, <3.11"
+requires-python = ">=3.9, <=3.11"
```

```suggestion
requires-python = ">=3.9, <3.12"
```
This should fix the problem.

---

The hatch runners didn't give a clear error, but a local 3.11 environment test gave the following error:
```
ERROR: Package 'lilio' requires a different Python: 3.11.5 not in '<=3.11,>=3.9'
```

---

😂 Thanks for the solution @BSchilperoort ! I never realized that `python3.11` would exclude `python3.11.x` 😮!